### PR TITLE
[TTS]update lr schedulers from per iter to per epoch for VITS 

### PR DESCRIPTION
**Pull request by WongLaw**
1. update lr schedulers from per iter to per epoch for VITS 
2. save and eval VITS per epoch

### CODE REVIEW

File: examples/csmsc/vits/conf/default.yaml [hunk 1]
```diff
@@ -179,7 +179,7 @@ generator_first: False # whether to start updating generator first
 #                OTHER TRAINING SETTING                  #
 ##########################################################
 num_snapshots: 10            # max number of snapshots to keep while training
-train_max_steps: 350000      # Number of training steps. == total_iters / ngpus, total_iters = 1000000
+max_epoch: 1000              # Number of training epochs.
```
Comment (yt605155624) [inline examples/csmsc/vits/conf/default.yaml:182]: 这里修改的原因？ train.py 也需要相应修改吧

File: paddlespeech/t2s/models/vits/vits_updater.py [hunk 2]
```diff
@@ -166,7 +166,8 @@ def update_core(self, batch):
                 gen_loss.backward()
 
                 self.optimizer_g.step()
-                self.scheduler_g.step()
+                if self.state.iteration % self.updates_per_epoch == 0:
```
Comment (yt605155624) [inline paddlespeech/t2s/models/vits/vits_updater.py:170]: 加个注释吧

File: examples/csmsc/vits/conf/default.yaml (hunk 1 above)
Comment (WongLaw) [reply examples/csmsc/vits/conf/default.yaml:182]: 手快漏加，done

File: paddlespeech/t2s/models/vits/vits_updater.py (hunk 2 above)
Comment (WongLaw) [reply paddlespeech/t2s/models/vits/vits_updater.py:170]: done

**Review by lizezheng — DISMISSED** [review]

**Review by yt605155624 — APPROVED** [review]
LGTM

**Review by zh794390558 — APPROVED** [review]
LGTM

**Pull request merged**